clodius/tiles/fasta.py

```diff
@@ -0,0 +1,260 @@
+from pyfaidx import Fasta
+import re
+import numpy as np
+import functools as ft
+import pandas as pd
+import logging
+import base64
+
+logger = logging.getLogger(__name__)
+
+TILE_SIZE = 1024
+
+
+def get_quadtree_depth(chromsizes):
+    tile_size_bp = TILE_SIZE
```

Comment:
Any reason to do this assignment instead of just using the global in the line below?

Reply:
This was copied from the other files. I didn't look into why, but it does seem extraneous.